src/main/resources/application.yaml

```diff
@@ -10,7 +10,9 @@ management:
       base-path: /
       exposure:
         include: health, info, prometheus
-
+  health:
+    case-document-am-api:
+      enabled: ${ENABLE_CDAM:true}
```

Comment:
default should be false ?

Reply:
I have set false in aat and prod using flux